services/lsst-dev.rst

```diff
@@ -256,6 +257,39 @@ Administrators may wish to note that the shared stack is automatically updated u
 
 .. _HTCondor pool: https://confluence.lsstcorp.org/display/DM/Orchestration
 
+.. _lsst-dev-testdata:
+
+Validation/Test Data Sets
+=========================
+
+As of early April 2019, there are two ``cron`` jobs that will update a set of validation data repositories and test data repositories.
+These updates will trigger in the 02:00 hour local at the LDF.
+In most cases, this will be a fairly straight forward ``git pull``, but if corruption is detected, the repository will be cloned afresh.
+
+Test data location is: ``/project/shared/data/test_data``
+
+Included test data repositories are::
+
+  testdata_jointcal
+  testdata_cfht
+  testdata_subaru
+  testdata_decam
+  ap_verify_testdata
+  ap_pipe_testdata
+  ci_hsc
+  afwdata
+
+Validation data location is: ``/project/shared/data/validation_data``
+
+Included validation data repositories are::
+
+  validation_data_hsc
+  validation_data_decam
+  validation_data_cfht
+
+These are maintained by the ``lsstsw`` user (this is the same user that curates the shared stack on the ``lsst-dev`` system).
+Please get in contact with a user with access to that account in case of problems.
```

Comment:
I'd drop the last line, since it's posing at least as many questions as it answers.

Maybe replace it with “ask in #dm-infrastructure”?